WordPress/Classes/ViewRelated/Notifications/Tools/NotificationCommentDetailCoordinator.swift

```diff
@@ -50,8 +53,9 @@ class NotificationCommentDetailCoordinator: NSObject {
                 return
             }
 
-            self.createViewController(comment: comment)
-            completion(self.viewController)
+            self.createViewController(comment: comment, completion: {
+                completion(self.viewController)
+            })
```

Comment:
Optional stylistic suggestion:

```suggestion
            self.createViewController(comment: comment) {
                completion(self.viewController)
            }
```